ble/GattClient.h

```diff
@@ -244,7 +249,11 @@ class GattClient {
      * Set up a callback for read response events.
      */
     void onDataRead(ReadCallback_t callback) {
-        onDataReadCallback = callback;
+        onDataReadCallbackChain.add(callback);
+    }
+
+    ReadCallbackChain_t& onDataRead() {
```

Comment:
please add a documentation header. it might be good to add a code-example showing its use